packaging/suse/amazon-ecs-init.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
```

Comment:
I have a minor preference for the service being called `ecs` (as it is on the Amazon Linux AMI) or `amazon-ecs`.  I don't know that customers would expect to type `-init`.